_data/home.yml

```diff
@@ -1,19 +1,19 @@
 cards:
   - title: "Grid modeling"
     icon: "fas fa-project-diagram"
-    description: "Powsybl provides a complete grid components modeling, which is fully editable and extendable."
+    description: "Powsybl provides a thourough framework for grid components modeling, which is fully editable and extendable."
```

Comment:
I prefer the former formula.

Reply:
I changed framework into library in the next commit.